task/anchore-cli/0.1/README.md

```diff
@@ -0,0 +1,81 @@
+# Anchore CLI
+
+The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+Anchore CLI will try to connect to the Anchore Engine at http://localhost/v1 with no authentication. The username, password and URL for the server can be passed to the Anchore CLI as command line arguments
```

Comment:
can we point to anchore documentation here ?

Reply:
done